.github/workflows/ci.yml

```diff
@@ -10,38 +10,39 @@ jobs:
     strategy:
       matrix:
         variant:
-          - 7.4
-          - 7.4-alpine
-          - 8.0
+          - "7.4"
+          - "7.4-alpine"
+          - "8.0"
         platform:
         - linux/amd64
-        # - linux/arm64
+        - linux/arm64
         exclude:
-          - props:
-              Dockerfile: Dockerfile-7.4-alpine
+          # TODO: enable with compile cache
           - platform: linux/arm64
     env:
-      TEST_MATCH: PHP Version ${{ matrix.props.version }}
+      TEST_MATCH: PHP Version ${{ matrix.variant }}
     steps:
       -
         name: Checkout
         uses: actions/checkout@v2
+      -
+        name: Install goss
+        run: |
+          curl -L https://github.com/aelsabbahy/goss/releases/download/v0.3.9/goss-linux-amd64 -o /usr/local/bin/goss
```

Comment:
@bryanlatten : It looks like you already account for it since you're excluding `linux/arm64` in a previous line above  (line 21 I think) but will we need to account for `arm64` if and when it comes? Or should we do the arch string selection in here or a script and since we're excluding arm64, the next time it does become available we only remove it from the exclude?

Reply:
@adobejmong yes, it gets super complicated at that point. Note: these instances of goss are *outside* the container, meaning they evaluate the assembled package. `dgoss` specifically pokes a hole in the container and mounts its own `goss` for testing. So, we will have to account for an x64 machine emulating a arm64 container run and poking/testing on that. Definitely for a followup (I welcome contribution!)